canvas_modules/common-canvas/src/common-properties/controls/textarea/textarea.jsx

```diff
@@ -98,7 +98,7 @@ class TextareaControl extends React.Component {
 
 		let display = textArea;
 		if (this.props.tableControl) {
-			const tooltipId = uuid4() + "-tooltip-column-" + this.props.propertyId.toString();
+			const tooltipId = uuid4() + "-tooltip-column-" + getPropertyIdOfTableControlString(this.props.propertyId);
```

Comment:
Before there was `[Object Object]` in tooltipId  -
![Screenshot 2023-03-16 at 5 28 31 PM](https://user-images.githubusercontent.com/25124000/225781706-f4744a1b-df94-4ada-850a-791f4ca4ef83.png)

After -
![Screenshot 2023-03-16 at 5 33 56 PM](https://user-images.githubusercontent.com/25124000/225782153-bd0ab899-4174-4401-b432-6ae06e4a9b04.png)

Reply:
Are we getting new uuid each time we rerender?  That doesn't seem like a good idea.  Should we move the uuid to the constructor so it's only generated once for this class?

Reply:
Yes there's a new `uuid` for each rerender and all table cells have a different `uuid`. I will generate `uuid` once in constructor and use it for all tooltipIds.